packages/account/src/Components/poi-poa-docs-submitted/poi-poa-docs-submitted.jsx

```diff
@@ -34,7 +34,8 @@ const PoiPoaDocsSubmitted = ({
         const { manual_status, poi_verified_for_vanuatu_maltainvest, poi_verified_for_bvi_labuan, poa_pending } =
             getAuthenticationStatusInfo(account_status);
         const is_vanuatu_or_maltainvest_selected =
-            jurisdiction_selected_shortcode === 'vanuatu' || jurisdiction_selected_shortcode === 'maltainvest';
+            jurisdiction_selected_shortcode === Jurisdiction.VANUATU ||
```

Comment:
We can refactor this as `[Jurisdiction.VANUATU,Jurisdiction.MALTA_INVEST].includes(jurisdiction_selected_shortcode)`